src/main/java/edu/harvard/iq/dataverse/settings/SettingsServiceBean.java

```diff
@@ -33,6 +33,7 @@
      * So there.
      */
     public enum Key {
+        IdentifierGenerationStyle,
```

Comment:
@landreev we should document `:IdentifierGenerationStyle` at http://guides.dataverse.org/en/4.6.1/installation/config.html#database-settings but for the next version, of course.